addons/token-exchange/agent_mirrorpeer_controller.go

```diff
@@ -50,14 +49,17 @@ type MirrorPeerReconciler struct {
 }
 
 const (
-	RookCSIEnableKey              = "CSI_ENABLE_OMAP_GENERATOR"
-	RookVolumeRepKey              = "CSI_ENABLE_VOLUME_REPLICATION"
-	MirroringModeKey              = "mirroringMode"
-	SchedulingIntervalKey         = "schedulingInterval"
-	ReplicationSecretNameKey      = "replication.storage.openshift.io/replication-secret-name"
-	ReplicationSecretNamespaceKey = "replication.storage.openshift.io/replication-secret-namespace"
-	ProvisionerTemplate           = "%s.rbd.csi.ceph.com"
-	RookConfigMapName             = "rook-ceph-operator-config"
+	RookCSIEnableKey                   = "CSI_ENABLE_OMAP_GENERATOR"
+	RookVolumeRepKey                   = "CSI_ENABLE_VOLUME_REPLICATION"
+	MirroringModeKey                   = "mirroringMode"
+	SchedulingIntervalKey              = "schedulingInterval"
+	ReplicationSecretNameKey           = "replication.storage.openshift.io/replication-secret-name"
+	ReplicationSecretNamespaceKey      = "replication.storage.openshift.io/replication-secret-namespace"
+	RBDProvisionerTemplate             = "%s.rbd.csi.ceph.com"
```

Comment:
where `RBDProvisionerTemplate` is set when creating the volume replication class?

Reply:
Good catch ! Got lost in the refactorings.
Added a testcase for this !